src/notifications/contracts/model-schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "to": "String",
+  "type": "String",
+  "title": "String",
+  "message": "String",
+  "unread": {
+    "type": "Boolean",
+    "default": true
+  },
+  "sent": {
+    "type": "Boolean",
+    "default": false
+  },
```

Comment:
Sure @tatianacmh, can we create a separate ticket for that? I just replicated the logic that was already in place, let me know if you want me to add this right away.